cmd/ocm-support/create/role_bindings/organization_role_binding/cmd.go

```diff
@@ -0,0 +1,64 @@
+package organization
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/account"
+	"github.com/openshift-online/ocm-support-cli/pkg/organization"
+	rolebinding "github.com/openshift-online/ocm-support-cli/pkg/role_binding"
+)
+
+// CmdCreateOrganizationRoleBinding represents the create organization role binding command
+var CmdCreateOrganizationRoleBinding = &cobra.Command{
+	Use:     "organizationRoleBinding [accountID] [roleID] [orgID]",
+	Aliases: utils.Aliases["organizationRoleBinding"],
+	Short:   "Assigns a role binding to an Account at organization level",
+	Long:    "Assigns a role binding to an Account at organization level",
+	RunE:    runCreateOrganizationRoleBinding,
+	Args:    cobra.ExactArgs(3),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		accountID := args[0]
+		roleID := args[1]
+		orgID := args[2]
```

Comment:
@tirthct according to our doc, roleID should be the last parameter. Please remember to update the readme as well.